### ISSUE TITLE
NullPointerException: null - getEffectColor

### ISSUE THREAD

**Issue by mibby** [elBukkit/MagicPlugin#908]
Magic dev 3725

https://paste.gg/p/anonymous/1515f200ad784ed2b3456566640c1c54/files/0e89db2fff5d4da2aa6bb5633e60e804/raw
```
[16:24:27] [Craft Scheduler Thread - 338 - Magic/ERROR]: [Magic] There was an error checking spell fire for issues
java.lang.NullPointerException: null
	at com.elmakers.mine.bukkit.magic.Mage.getEffectColor(Mage.java:2478) ~[?:?]
	at com.elmakers.mine.bukkit.action.CastContext.getEffectColor(CastContext.java:632) ~[?:?]
	at com.elmakers.mine.bukkit.action.CastContext.getEffects(CastContext.java:619) ~[?:?]
	at com.elmakers.mine.bukkit.action.builtin.CustomProjectileAction.prepare(CustomProjectileAction.java:280) ~[?:?]
	at com.elmakers.mine.bukkit.action.ActionContext.prepare(ActionContext.java:28) ~[?:?]
	at com.elmakers.mine.bukkit.action.ActionHandler.prepare(ActionHandler.java:132) ~[?:?]
	at com.elmakers.mine.bukkit.spell.ActionSpell.reloadParameters(ActionSpell.java:175) ~[?:?]
	at com.elmakers.mine.bukkit.tasks.ValidateSpellsTask.run(ValidateSpellsTask.java:47) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:100) ~[patched_1.16.5.jar:git-Airplane-"1cf4e79"]
	at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:58) ~[patched_1.16.5.jar:git-Airplane-"1cf4e79"]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.16.5.jar:git-Airplane-"1cf4e79"]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
```

**Comment by NathanWolf** [comment]
Thank you- looks like this is an issue if you're not using classes. I have a fix in the latest dev and stable builds:

https://jenkins.elmakers.com/job/Magic-Stable/31/

https://jenkins.elmakers.com/job/MagicPlugin/3726/

**Comment by NathanWolf** [comment]
Released in 8.7.1, will also be fixed in 9.0
